commit 5e7c7f482079b10a2f48720c6ca51e28df224715
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Thu Aug 27 15:40:29 2009 +0000

    Add VA_STATUS_ERROR_UNIMPLEMENTED string representation.

commit f5661a9056ed8ab9110267f08b8606af98bfdc57
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Wed Aug 26 16:01:10 2009 +0000

    Add VA_STATUS_ERROR_UNIMPLEMENTED.

diff --git a/src/va.c b/src/va.c
index 9300291..3f4975a 100644
--- a/src/va.c
+++ b/src/va.c
@@ -350,6 +350,8 @@ const char *vaErrorStr(VAStatus error_status)
             return "invalid parameter";
         case VA_STATUS_ERROR_RESOLUTION_NOT_SUPPORTED:
             return "resolution not supported";
+        case VA_STATUS_ERROR_UNIMPLEMENTED:
+            return "the requested function is not implemented";
         case VA_STATUS_ERROR_UNKNOWN:
             return "unknown libva error";
     }
diff --git a/src/va.h b/src/va.h
index 6b4bbd4..81bc641 100755
--- a/src/va.h
+++ b/src/va.h
@@ -125,6 +125,7 @@ typedef int VAStatus;	/* Return status type from functions */
 #define VA_STATUS_ERROR_FLAG_NOT_SUPPORTED      0x00000011
 #define VA_STATUS_ERROR_INVALID_PARAMETER	0x00000012
 #define VA_STATUS_ERROR_RESOLUTION_NOT_SUPPORTED 0x00000013
+#define VA_STATUS_ERROR_UNIMPLEMENTED           0x00000014
 #define VA_STATUS_ERROR_UNKNOWN			0xFFFFFFFF
 
 /* De-interlacing flags for vaPutSurface() */
